pkg/security/resolvers/cgroup/model/model.go

```diff
@@ -98,3 +99,8 @@ func (cgce *CacheEntry) SetTags(tags []string) {
 func (cgce *CacheEntry) NeedsTagsResolution() bool {
 	return len(cgce.ID) != 0 && !cgce.WorkloadSelector.IsEmpty()
 }
+
+// GetCreationTime returns the root pid of a cgroup
+func (cgce *CacheEntry) GetCreationTime() uint64 {
```

Comment:
Nit: why a Get function when the underlying property is already public ?

Reply:
Because I was too lazy to remove the function when figuring out this is useless :-)
I removed it